testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -12,6 +12,7 @@ Feature: Create activation keys
   Scenario: Create an activation key with a channel
     When I follow the left menu "Systems > Activation Keys"
     And I follow "Create Key"
+    And I wait until I do not see "Loading..." text
```

Comment:
We really need to check including dots?
I think we can be more flexible in our checks if we don't add the final dots, three dots, etc.
In any case... approving the PR.

Reply:
I think it is better to add them. What happens if somebody add "Loading" as part of the description?
Or we add a text on this page which contain the test "Loading".
I just feel better when we have it.